lib/git-subrepo

```diff
@@ -694,6 +690,16 @@ subrepo:fetch() {
     error "Can't fetch subrepo. Remote is 'none' in '$subdir/.gitrepo'."
   fi
 
+  if $all_wanted && [[ -n $subrepo_branch ]]; then
```

Comment:
Wouldn't something similar be necessary in subrepo:push?

Reply:
I'm not sure. I don't think anything is missing in subrepo:push. It's works fine with --all.